rest_sensors.cpp

```diff
@@ -1751,6 +1751,56 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                         rspItemState[QString("Error : unknown Window open setting for %1").arg(sensor->modelId())] = map[pi.key()];
                     }
                 }
+                else if (rid.suffix == RConfigSwingMode)
+                {
+                    if (map[pi.key()].type() == QVariant::String && map[pi.key()].toString().size() <= 19)
+                    {
+                        if (sensor->modelId() == QLatin1String("AC201"))
+                        {
+                            QString modeSet = map[pi.key()].toString();
+                            quint8 mode = 0;
+
+                            if (modeSet == "fully closed") { mode = 0x01; }
+                            else if (modeSet == "fully open") { mode = 0x02; }
+                            else if (modeSet == "quarter open") { mode = 0x03; }
+                            else if (modeSet == "half open") { mode = 0x04; }
+                            else if (modeSet == "three quarters open") { mode = 0x05; }
+                            else
+                            {
+                                rspItemState[QString("error unknown swing mode for %1").arg(sensor->modelId())] = map[pi.key()];
```

Comment:
This doesn't look right, errors shouldn't be reported in `rspItemState`?

Reply:
Might be, but that is all over the place...

Reply:
The error style was now corrected in the other PR, could this here be aligned too?

Reply:
This should actually be covered by #4493